src/music.cpp

```diff
@@ -0,0 +1,93 @@
+#include "music.h"
+#include <fstream>
+#include <iostream>
+
+Asound::Asound() {
+  std::ifstream file(MUSIC_CONFIG_PATH + "config.json");
+
+  if (!file.is_open()) {
+    std::cerr << "Failed to open config file for music" << std::endl;
+    exit(1);
+  }
+
+  file >> config;
+
+  for (auto &[k, v] : config.items()) {
+    std::cout << "Read sound key: " << k << " val: " << v << std::endl;
+    std::string s = MUSIC_CONFIG_PATH + std::string(v);
+    sounds.emplace(k, s);
+  }
+}
+
+Asound::~Asound() {}
+
+raylib::Sound &Asound::play(const std::string sound) {
+  return (*this)[sound].Play();
+}
+
+raylib::Sound &Asound::pause(const std::string &sound) {
```

Comment:
I think wrapping all `raylib::Sound` methods other than `play` is confusing. just have `play` and `operator[]` and we can call other methods like pause directly

Reply:
it doesn't hurt to keep it right? thoughts, @jacobrec?